contrib/win32/openssh/OpenSSHTestHelper.psm1

```diff
@@ -412,7 +412,7 @@ function Run-OpenSSHE2ETest
    # Discover all CI tests and run them.
     Push-Location $Script:E2ETestDirectory
     Write-Log -Message "Running OpenSSH E2E tests..."    
-    $testFolders = Get-ChildItem *.tests.ps1 -Recurse -Exclude SSHDConfig.tests.ps1, SSH.Tests.ps1 | ForEach-Object{ Split-Path $_.FullName} | Sort-Object -Unique
```

Comment:
Note that the exclude has no impact since this line is to get all the test folders. so I removed the exclude. To disable tests in CI, we need to change the tag to other value than CI.